pom.xml

```diff
@@ -234,6 +235,13 @@
                             	<exclude>WEB-INF/lib/log4j-*-2.11.*.jar</exclude>  
                             	<exclude>WEB-INF/lib/log4j-1.2-api-2.11.0.jar</exclude>
                             	<exclude>WEB-INF/lib/httpclient-4.5.5.jar</exclude>
+                            	<exclude>WEB-INF/lib/log4j-1.2-api-2.13.2.jar</exclude>
```

Comment:
When I build this branch locally, I'm seeing the Log4J 2.13.3 JARs in the exploded WAR file. I would suggest updating these WAR exclusions to match "2.13.*" instead of "2.13.2".

Reply:
ok I will set them to 2.13.*